ui-new/html/deposit-timeout.html

```diff
@@ -0,0 +1,16 @@
+<section class="viewport deposit_timeout_state cash-out-color" data-tr-section="deposit-timeout">
+  <section class="mainv">
+    <section class="header">
+      <img src="images/nothing-yet.svg"/>
+      <h3 class="js-i18n">Nothing yet.</h3>
+    </section>
+    <section class="info">
+      <p class="js-i18n d2">No coins are showing up.</p>
+      <p class="js-i18n d2b">Have you sent the BTC yet?</p>
+    </section>
+    <section class="actions">
+      <span class="button-yes js-i18n" id="deposit-timeout-sent-yes">Yes, I have</span>
```

Comment:
These buttons won't work, if you check out app.js there is a function called `targetButton` that travels the dom node looking for specific classes before stopping, you should add a `button` class here and on all buttons